### PR TITLE
Jetson docker

### PR DESCRIPTION
**Pull request by borisfom**
Here's for your review: 
The only change (besides scripts/Docker that I have added) is in requirements.txt (and test.txt). I had to change onnxruntime req to onnxruntime_gpu_tensorrt.   
Unfortunately, unlike .deb dependencies, there is no way to specify ‘or’ condition in requirements.txt. I would like to explore what we can do here - same issue will be encountered by x86 users who would use ONNX runtime built with TRT backend.
One solution is to get rid of onnxruntime in requirements.txt altogether. I could obviously create alternative requirement_jetson.txt and  use copy command in Dockerfile - however, as I pointed out above, this may not be Jetson-specific issue.
We can’t change to onnxruntime_gpu_tensorrt either - this package is not found by pip on x86
We could change x86 Dockerfile though to build onnxruntime_gpu_tensorrt, same way I did on Jetson. 
That may depend on our plan for future use of onnxruntime with Nemo - and if we care about CPU-only users.  If it’s going to stay marginal like it is now, or CPU users are more important, then we should stick to using stock onnxruntime module for x86 and have separate requirements for Jetson. If we foresee more use of onnxruntime in NeMo, it’s better to make sure it’s being used efficiently  - built with GPU/TRT. Either way is not too hard - I already have onnxruntime_gpu_tensorrt  build script - it should be usable unchanged with x86.

### CODE REVIEW

**Comment by drnikolaev** [comment]
I totally support this idea. Two engines (ORT and TRT) would help us to compare results numerically as well as performance-wise.

**Comment by borisfom** [comment]
> I totally support this idea. Two engines (ORT and TRT) would help us to compare results numerically as well as performance-wise.

Thanks Sergey!
@okuchaiev : let me add the change to this PR to include building ONNXruntime with TRT support in regular Dockerfile as well.

File: requirements/requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 numpy>=1.18.2
 onnx
-onnxruntime
+onnxruntime_gpu_tensorrt
```
Comment (tkornuta-nvidia) [inline requirements/requirements.txt:3]: Hey @borisfom can you please make sure the dependency on `onnxruntime_gpu_tensorrt` won't break installing/running NeMo applications/tests  on machines WITHOUT GPUs?

Comment (tkornuta-nvidia) [reply requirements/requirements.txt:3]: This is what I have when I pull your PR and try to run ./reinstall.sh on my MacBook:

```
Requirement already satisfied: numpy>=1.18.2 in /Users/tkornuta/opt/anaconda3/envs/nemo-env/lib/python3.7/site-packages (from nemo-toolkit==0.11.0b5) (1.18.4)
Requirement already satisfied: onnx in /Users/tkornuta/opt/anaconda3/envs/nemo-env/lib/python3.7/site-packages/onnx-1.6.0-py3.7-macosx-10.7-x86_64.egg (from nemo-toolkit==0.11.0b5) (1.6.0)
ERROR: Could not find a version that satisfies the requirement onnxruntime_gpu_tensorrt (from nemo-toolkit==0.11.0b5) (from versions: none)
ERROR: No matching distribution found for onnxruntime_gpu_tensorrt (from nemo-toolkit==0.11.0b5)
```

Comment (borisfom) [reply requirements/requirements.txt]: Correct - I am working right now to change the version we are building to _gpu (no TRT) and to add onnxruntime build step to x86 build scripts.

**Comment by borisfom** [comment]
I have pushed changes to build onnxruntime_gpu on ARM and to use PIP version on Intel.

**Comment by okuchaiev** [comment]
please fix code style like this: ``python setup.py style --fix`` - the CI is failing on it

**Comment by borisfom** [comment]
> please fix code style like this: `python setup.py style --fix` - the CI is failing on it

Done - though that typo was in master. Also reverted notebook changes I did not mean.

**Comment by borisfom** [comment]
Superseded by https://github.com/NVIDIA/NeMo/pull/765

**Pull request closed without merging**